shop/src/main/java/ro/msg/learning/shop/model/Location.java

```diff
@@ -0,0 +1,24 @@
+package ro.msg.learning.shop.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+@Entity
+@Table(name="LOCATION")
+public class Location extends BaseEntity{
+    private String name;
+    private String addressCountry;
```

Comment:
Within the Customer entity you used snake case but here you used camel case, I suggest switching everywhere to snake case because Hibernate will always convert them to snake case when doing the mappings